pytensor/tensor/fft.py

```diff
@@ -39,9 +39,10 @@ def perform(self, node, inputs, output_storage):
         a = inputs[0]
         s = inputs[1]
 
+        # FIXME: This call is deprecated in numpy 2.0
```

Comment:
will this not cause problems in the very near future?

Reply:
When we remove the pin to numpy 2.0. It's how of scope of this PR, and we have way bigger problems with numpy 2.0